.stylelintrc.json

```diff
@@ -6,6 +6,10 @@
       {
         "ignore": "local"
       }
-    ]
+    ],
+    "function-whitespace-after": null,
+    "media-feature-range-operator-space-after" : null,
+    "media-feature-range-operator-space-before" : null,
+    "string-quotes": null
```

Comment:
I believe `stylelint-config-bitcrowd` might be outdated. 
I added these changes to fix this lint error: 

<img width="1089" alt="Screenshot 2023-08-01 at 14 12 36" src="https://github.com/bitcrowd/bitstyles/assets/17965508/91d98e59-229a-4446-bdc7-4c01232d0e8f">

Reply:
Yes it needs to be updated for v15 of stylelint :+1: I added an issue over on our linting repo: https://github.com/bitcrowd/frontend/issues/39